src/components/CheckoutActions/CheckoutActions.js

```diff
@@ -91,6 +146,16 @@ export default class CheckoutActions extends Component {
       };
     }
 
+    // Order summary
+    const { fulfillmentTotal, itemTotal, taxTotal, total } = summary;
+    const checkoutSummary = {
+      displayShipping: fulfillmentTotal && fulfillmentTotal.displayAmount,
+      displaySubtotal: itemTotal && itemTotal.displayAmount,
+      displayTotal: total && total.displayAmount,
+      displayTax: taxTotal && taxTotal.displayAmount,
+      items
```

Comment:
The `fulfillmentTotal && ` and such are not necessary because they are all required fields in GraphQL schema.